pkg/odo/cli/component/create.go

```diff
@@ -71,11 +71,11 @@ const CreateRecommendedCommandName = "create"
 // since the application will always be in the same directory as `.odo`, we will always set this as: ./
 const LocalDirectoryDefaultLocation = "./"
 
-// DevfilePath is the default path of devfile.yaml
-const DevfilePath = "./devfile.yaml"
+// DevfilePath is the path of devfile.yaml, the default path is "./devfile.yaml"
+var DevfilePath = filepath.Join(LocalDirectoryDefaultLocation, "devfile.yaml")
 
-// EnvFilePath is the default path of env.yaml for devfile component
-const EnvFilePath = "./.odo/env/env.yaml"
+// EnvFilePath is the path of env.yaml for devfile component, the defult path is "./.odo/env/env.yaml"
+var EnvFilePath = filepath.Join(LocalDirectoryDefaultLocation, ".odo/env/env.yaml")
```

Comment:
This just got me thinking, are any of these directories safe to use on Windows, as we're using Unix paths.

@jgwest thoughts?

Reply:
From the `filepath` docs https://golang.org/pkg/path/filepath/#Join, I think `filepath.Join` can handle the path conversion to the target OS

Reply:
Yah, As Jingfu mentioned, the `filepath` package works like Java's `File` class and allows forward slash on Windows (with the API translating as needed behind the scenes.)